internal/store/engine_state.go

```diff
@@ -458,5 +453,9 @@ func StateToView(s EngineState) view.View {
 
 	ret.Log = string(s.Log)
 
+	if s.LastTiltfileError != nil {
+		ret.TiltfileErrorMessage = s.LastTiltfileError.Error()
```

Comment:
does this get the right error message if there's a stack trace?

Reply:
Oooh good call it doesn't!